dune-project

```diff
@@ -23,7 +23,9 @@
  (description "A project generator for Reason and OCaml\n")
  (depends
   (ocaml
-   (and (>= 4.08.0) (< 4.11.0)))
+   (and
+    (>= 4.08.0)
+    (< 4.11.0)))
```

Comment:
Do you still need the upper bound on 4.11?

Reply:
Appreciate the review, thanks a lot! I removed the upper bound 🙂